mix.exs

```diff
@@ -134,7 +134,6 @@ defmodule Trento.MixProject do
   defp aliases do
     [
       start: [
-        "cmd docker-compose up -d",
```

Comment:
Let's keep this command out. We don't have to bind the start command to a specific local environment

Reply:
You are right, forget my comment